debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDdlParserTest.java

```diff
@@ -654,31 +656,31 @@ public void shouldParseSetOfSessionVariable() {
 
     @Test
     public void shouldParseButNotSetUserVariableWithHyphenDelimiter() {
-        String ddl = "SET @a-b-c-d:=1";
+        String ddl = "SET @a_b_c_d:=1";
```

Comment:
Why is this changing? Esp. it doesn't match the method name any longer.

Reply:
Again, according to the grammar, the @a-b-c-d is not a valid name for a custom variable or better said variable with this name has to be in quotes because the hyphen delimiter is used for something else.